style/samples/ObjectiveC.m

```diff
@@ -49,8 +46,8 @@ - (id)initWithCoder:(NSCoder *)aDecoder
 - (void)shuffleCards
 {
     // Objective-C literals are your friend
-    NSDictionary *themeColors = @{ kRedColor : [UIColor redColor], kBlueColor : [UIColor blueColor] };
-    NSArray *robots = @[ @"Ralph", @"Bender", @"The Iron Giant" ];
+    NSDictionary *themeColors = @{redColor: [UIColor redColor], blueColor: [UIColor blueColor]};
```

Comment:
I think it should be `BlueColor`, not `blueColor`, for the record. I didn't make that clear enough earlier.